src/main/resources/META-INF/rewrite/was-to-liberty.yml

```diff
@@ -26,21 +26,22 @@ description: >
 recipeList:
   - org.openrewrite.java.liberty.RemoveWas2LibertyNonPortableJndiLookup
   - org.openrewrite.java.liberty.ServerName
-  - org.openrewrite.java.liberty.WebSphereUnavailableSSOCookieMethod
-  - org.openrewrite.java.liberty.WebSphereUnavailableSSOTokenMethod
+  - org.openrewrite.java.liberty.WebSphereUnavailableSSOMethods
   - org.openrewrite.xml.liberty.AppDDNamespaceRule
   - org.openrewrite.xml.liberty.ConnectorDDNamespaceRule
   - org.openrewrite.xml.liberty.EJBDDNamespaceRule
   - org.openrewrite.xml.liberty.PersistenceXmlLocationRule
   - org.openrewrite.xml.liberty.WebDDNamespaceRule
+  - org.openrewrite.xml.liberty.DetectUnsupportedJSF
 ---
 type: specs.openrewrite.org/v1beta/recipe
-name: org.openrewrite.java.liberty.WebSphereUnavailableSSOTokenMethod
-displayName: Use `getSSOCookieFromSSOToken`
+name: org.openrewrite.java.liberty.WebSphereUnavailableSSOMethods
+displayName: Use `getSSOCookieFromSSOToken` and `logout`
 description: >
-  This method `LTPACookieFromSSOToken()` is deprecated in traditional WebSphere Application 
-  Server Version 8.5 and might be removed in a future release. It is not available on Liberty.
+  This recipe replaces`LTPACookieFromSSOToken()` with  `getSSOCookieFromSSOToken`  and `revokeSSOCookies` with `logout`. The two methods are  deprecated in traditional WebSphere Application 
```

Comment:
We need to add a space between `replacesLTPACookieFromSSOToken()`. Aside from that everything else looks good

Reply:
added in next commit

Reply:
Thanks!